lib/rules/require-description.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * @author Yosuke Ota <https://github.com/ota-meshi>
+ * See LICENSE file in root directory for full license.
+ */
+"use strict"
+
+const utils = require("../internal/utils")
+
+module.exports = {
+    meta: {
+        docs: {
+            description:
+                "require include descriptions in ESLint directive-comments",
+            category: "Stylistic Issues",
+            recommended: false,
+            url:
+                "https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/require-description.html",
+        },
+        fixable: null,
+        schema: [
+            {
+                type: "object",
+                properties: {
+                    ignore: {
+                        type: "array",
+                        items: {
+                            enum: [
+                                "eslint",
+                                "eslint-disable",
+                                "eslint-disable-line",
+                                "eslint-disable-next-line",
+                                "eslint-enable",
+                                "eslint-env",
+                                "exported",
+                                "global",
+                                "globals",
+                            ],
+                        },
+                        additionalItems: false,
+                        uniqueItems: true,
+                    },
```

Comment:
Since this array/enum definition is used by multiple rules now, might be nice to move it to a shared constant?

```suggestion
                    ignore: directiveTypes,
```